docs/source/tutorials.rst

```diff
@@ -0,0 +1,21 @@
+.. _tutorials:
+
+=========================
+Useful Vivarium Tutorials
+=========================
+
+Each Vivarium repository contains documentation in the form of API references,
+tutorials, and/or conceptual information. These tutorials are especially useful
+and explain many common tasks but because they are scattered across many
+repositories they can be hard to find. Below is a list of some of the most
+important pieces of documentation to know about.
+
+
+- `The model specification <https://vivarium.readthedocs.io/projects/vivarium/en/latest/concepts/model_specification.html>`_
+- `Running a simulation <https://vivarium.readthedocs.io/projects/vivarium/en/latest/tutorials/running_a_simulation/index.html#running-a-simulation>`_
+- `Exploring a simulation in an interactive setting <https://vivarium.readthedocs.io/projects/vivarium/en/latest/tutorials/exploration.html#exploring-a-simulation-in-an-interactive-setting>`_
+- `Creating and altering a data artifact <https://vivarium-public-health.readthedocs.io/en/latest/tutorials/artifact.html#artifact>`_
+- `Pulling data using Vivarium Inputs <https://vivarium-inputs.readthedocs.io/en/latest/tutorials/pulling_data.html>`_
+- `The Vivarium Inputs data-pulling function API <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/api_reference/interface.html>`_
```

Comment:
do we need to link to the API docs here? The tutorial covers how to use all of the functions in the interface so it seems redundant. Plus (being pedantic) but the API docs aren't technically a tutorial...

Reply:
I thought I removed that bc the tutorial has the tables I was interested in including